docs/docs/command-line-flags.md

```diff
@@ -27,6 +27,9 @@ The following options can influence how Marathon works:
     Examples: `"hdfs://localhost:54310/path/to/store"`,
     `"file:///var/log/store"`. For details, see the
     [artifact store]({{ site.baseurl }}/docs/artifact-store.html) docs.
+* `--access_control_allow_origin` (Optional. Default: None):
+    The origin(s) to allow in Marathon. Not set by default.
```

Comment:
How about: "Comma separated list of allowed originating domains for HTTP requests."